index.html

```diff
@@ -1,11 +1,757 @@
 <!DOCTYPE html>
-  <html>
-      <head>
-        <title>NY Times Clone</title>
-        <link href="styles.css" rel="stylesheet" type="text/css">
-        <meta charset="utf-8">
-      </head>
-      <body>
-        
-      </body>
-  </html>
\ No newline at end of file
+<html>
+  <head>
+    <title>NY Times Clone</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <meta charset="utf-8">
+  </head>
+  <body>
+    <main>
+      <div class="container">
+        <header>
```

Comment:
`REQUIRED`

This should be a `<nav>` element rather than `<header>`. The content of this section is are the navigation menu and links.